java/src/jmri/jmrit/logix/Warrant.java

```diff
@@ -1927,7 +1927,8 @@ private boolean restoreRunning(String speedType) {
             TrainOrder to = bo.allocatePaths(this, true);
             OBlock block = bo.getBlock();
             if (log.isDebugEnabled()) {
-                log.debug("{}: {}", getDisplayName(), to.toString());
+                log.debug("{}: {}", getDisplayName(), to.toString());   // spotbugs doesn't like this
```

Comment:
I think the problem with this line is that `.toString()` is not needed in `to.toString()`.

Try to replace it with:
`log.debug("{}: {}", getDisplayName(), to);`